config.md

```diff
@@ -1,6 +1,6 @@
 # <a name="containerConfigurationFile" />Container Configuration file
 
-This configuration file contains metadata necessary to implement standard operations against the container.
+This configuration file contains metadata necessary to implement standard operations against the creation of a container.
```

Comment:
`process.args` is used by `start` and some hooks are used by `delete`.  But `kill` doesn't need anything from then config.  So we can draw a distinction here, but your current wording isn't quite it.

I'd rather link "standard operations" to the `runtime.md` section and maybe weaken it to "some standard operations".  But I'd avoid calling out creation without also mentioning the other impacted operations.